network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1044,6 +1133,14 @@ def exec(self, only_captured):
         return QueryAnswer(False, 'The connections allowed by ' + self.name1 +
                            ' do not intersect the connections allowed by ' + self.name2)
 
+    def compute_query_output(self, query_answer, cmd_line_flag=False):
+        if query_answer.bool_result:
+            query_output = self.config2.name + ' does not forbid connections specified in ' + \
+                                        self.config1.name + ':' + query_answer.output_explanation
+        else:
+            query_output = self.config2.name + ' forbids connections specified in ' + self.config1.name
+        return QueryOutput(query_answer.bool_result, query_output)
+
```

Comment:
Let's have a separate `ForbidsQuery` class, which calls `IntersectsQuery`. The same way as `PermitsQuery` calls `ContainmentQuery`.